bedrock/base/templates/macros.html

```diff
@@ -205,22 +205,24 @@
       <div class="triangle-border"></div>
 
       <ul class="adjunctnav{% if activenav == 'desktop' %} active{% endif %}">
+        <li><a href="{{ url('firefox.channel.desktop') }}" data-link-type="nav" data-link-name="Side Menu: Pre-release downloads">{{ _('Pre-release downloads') }}</a>
```

Comment:
We need to put it in main.lang, so no tags. I'll expose the strings later today.